regression/benchmark/benchmarks.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: MPL-2.0
+set -e
+
+# Ensure all dependencies are installed
+command -v jq >/dev/null 2>&1 || { echo >&2 "jq is not installed. Aborting."; exit 1; }
+command -v gcc >/dev/null 2>&1 || { echo >&2 "gcc is not installed. Aborting."; exit 1; }
+
+# Script directory
+BENCHMARK_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+
+prepare_benchmark() {
+    echo "Preparing benchmark environment..."
+    mkdir -p "/benchmark"
+
+    # Copy sysbench
+    echo "Copying sysbench to /benchmark..."
+    cp /usr/local/benchmark/sysbench/bin/sysbench /benchmark
+
+    # Compile getpid
+    echo "Compiling getpid..."
+    pushd "${BENCHMARK_DIR}/../apps/getpid" >/dev/null
+    gcc getpid.c -o /benchmark/getpid
+    popd >/dev/null
+}
+
+run_benchmark() {
+    local benchmark="$1"
+    local avg_pattern="$2"
+    local avg_field="$3"
+    local enable_kvm="$4"
+
+    local linux_output="${BENCHMARK_DIR}/linux_output.txt"
+    local aster_output="${BENCHMARK_DIR}/aster_output.txt"
+    local result_template="${BENCHMARK_DIR}/result_template_${benchmark}.json"
+    local result_file="result_${benchmark}.json"
+    local bench_script="${BENCHMARK_DIR}/${benchmark}.sh"
+    local kernel_dir="${BENCHMARK_DIR}/../build/linux-5.15.0"
+    if [ ! -d "${kernel_dir}" ]; then
+        mkdir -p "${kernel_dir}"
+    fi
+    local linux_kernel="${kernel_dir}/vmlinuz-5.15.0-105-generic"
```

Comment:
Update the Dockerfile to include this kernel image.

```
        curl -L -o "${linux_kernel}" \
            -H "Accept: application/vnd.github.v3.raw" \
            "https://api.github.com/repos/grief8/storage/contents/vmlinuz-5.15.0-105-generic"
```

Use an official source to download the image. Not your own repository.